docs/HISTORY.txt

```diff
@@ -5,7 +5,12 @@ Changelog
 1.4.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- BREAKING CHANGE: Disallow setting a custom permission at widget init time.
+  [Rotonen]
```

Comment:
Isn't this a frequently used use-case? I.e. allow new `labels` for editors and allow new `keywords` (for admin only).

We do not use custom-permissions in any of our projects. So I'm just thinking for future projects... 🤔

Reply:
This should get reintroduced once we figure out the permission bleeds.

For now we do not have a functional architecture to support this.